crest.md

```diff
@@ -53,3 +61,32 @@ In order to provide a seamless user experience, _CREST_ exposes a number of even
     console.log("Event listeners ready...");
 </script>
 ```
+
+## Language selector
+
+Language selectors are crucial for localised websites to ensure that the translations are prominently featured on the website. Crest is compatible with two types of language selector - a dropdown or a sidebar.
+
+### Sidebar
+
+The sidebar language selector is the easiest to integrate. It is part of the loader script, `stub.js`, so no further set-up is needed. It'll insert the flag of available languages to the side for users to click. This option offers minimal configuration. The following options are available:
+
+- The flags of each language.
+- The name of the language(s) as displayed when hovered.
+- The side the selector should show up.
+
+### Dropdown
+
+For websites where more customisation options are required, the dropdown language selector is available. Setup is slightly more involved:
+
+1. Create and empty `span` or `div` with the class `el-dropdown` (or the one configured on the Dashboard).
+2. Add the translator stub with the `?disableSelector` query set to `true` or edit the selector to include this query.
```

Comment:
I really need to _disable_ the selector to insert the dropdown?!

Reply:
You *should*. Otherwise, you get two selectors.